frappe/database/mariadb/database.py

```diff
@@ -123,6 +125,19 @@ def is_type_number(code):
 	def is_type_datetime(code):
 		return code in (pymysql.DATE, pymysql.DATETIME)
 
+	def rename_table(self, old_name: str, new_name: str) -> Union[List,Tuple]:
+		old_name = self.add_tab(old_name)
+		new_name = self.add_tab(new_name)
+		return self.sql(f"RENAME TABLE `{old_name}` TO `{new_name}`")
+
+	def DESC(self, doctype: str) -> Union[List,Tuple]:
```

Comment:
```suggestion
	def describe(self, doctype: str) -> Union[List,Tuple]:
```

Let's call this describe to avoid confusion between `frappe.db.DESC` and `frappe.qb.desc`